lib/perl/Genome/Model/ClinSeq.pm

```diff
@@ -645,33 +645,18 @@ sub _resolve_workflow_for_build {
     }
 
     #TophatJunctionsAbsolute - Run tophat junctions absolute analysis on normal
-    if ($build->normal_rnaseq_build) {
-        my $normal_tophat_junctions_absolute_op = $self->tophat_junctions_absolute_op($workflow, 'normal');
-    }
-    #TophatJunctionsAbsolute - Run tophat junctions absolute analysis on tumor
-    my $tumor_tophat_junctions_absolute_op;
-    if ($build->tumor_rnaseq_build) {
-        $tumor_tophat_junctions_absolute_op = $self->tophat_junctions_absolute_op($workflow, 'tumor');
-    }
-
     #CufflinksExpressionAbsolute - Run cufflinks expression absolute analysis on normal
     if ($build->normal_rnaseq_build) {
+        my $normal_tophat_junctions_absolute_op = $self->tophat_junctions_absolute_op($workflow, 'normal');
```

Comment:
Yeah, I'm only suggesting that, since the return values are not used for anything, assigning them to `my` variables isn't necessary.